sh/tests/input-redirection_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS='
+'
+
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    $(bash "$script_dirS"/student/input-redirection.sh)
+    submitted=$(bash "$script_dirS"/show_info.sh)
+    rm show_info.sh
+    $(bash "$script_dirS"/solutions/input-redirection.sh)
+    expected=$(bash "$script_dirS"/show_info.sh)
+    rm show_info.sh
+
+    diff <(echo "$submitted") <(echo "$expected")
+}
+
+challenge
```

Comment:
Just a general comment: the function `challenge` might be useful to run additional tests, but in this case, since there is no input argument, I don't see any advantage of using `challenge` vs running directly the code

Reply:
I agree... The rational here would be that almost all tests have `challenge` and when you create a new exercise you use a boilerplate, which will have `challenge`.
Keeping it results in:
- Less effort (you don't need to remove it and maybe refactor the body accordingly)
- More readability (people used to the `scripting pisicne` structure, will find something they already know and they can guess behavior)
- Ready for future improvements (for example if somebody decide to add one more challenge, changing before the value of the tested env variables it will be way easier with challenge already in place)